packages/braintree-utils/src/braintree-script-loader.ts

```diff
@@ -63,6 +64,18 @@ export default class BraintreeScriptLoader {
         return this.braintreeHostWindow.braintree.connect;
     }
 
+    async loadFastlane(): Promise<BraintreeFastlaneCreator> {
+        await this.scriptLoader.loadScript(
+            `//js.braintreegateway.com/web/${this.braintreeSdkVersion}/js/fastlane.min.js`,
```

Comment:
As far as I remember Fastlane will not be available for `this.braintreeSdkVersion` (I didn't see a change of braintree sdk version). Please double check the documentation

Reply:
changed version